docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -21,50 +29,85 @@ PUT /my_index
     }
 }
 
---------------------------------------------------
+----
 // CONSOLE
 // TESTSETUP
 
-And here is a sample nested query usage:
+[[nested-query-ex-query]]
+===== Example query
 
 [source,js]
---------------------------------------------------
-GET /_search
+----
+GET /my_index/_search
 {
     "query": {
         "nested" : {
             "path" : "obj1",
-            "score_mode" : "avg",
             "query" : {
                 "bool" : {
                     "must" : [
                     { "match" : {"obj1.name" : "blue"} },
                     { "range" : {"obj1.count" : {"gt" : 5}} }
                     ]
                 }
-            }
+            },
+            "score_mode" : "avg"
         }
     }
 }
---------------------------------------------------
+----
 // CONSOLE
 
-The query `path` points to the nested object path, and the `query`
-includes the query that will run on the nested docs matching the
-direct path, and joining with the root parent docs. Note that any
-fields referenced inside the query must use the complete path (fully
-qualified).
-
-The `score_mode` allows to set how inner children matching affects
-scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
-`max` and `none`.
-
-There is also an `ignore_unmapped` option which, when set to `true` will
-ignore an unmapped `path` and will not match any documents for this query.
-This can be useful when querying multiple indexes which might have different
-mappings. When set to `false` (the default value) the query will throw an
-exception if the `path` is not mapped.
-
-Multi level nesting is automatically supported, and detected, resulting
-in an inner nested query to automatically match the relevant nesting
-level (and not root) if it exists within another nested query.
+[[nested-top-level-params]]
+==== Top-level parameters for `nested`
+
+`path` (Required)::
+(string) Path to the nested object you wish to search.
+
+`query` (Required)::
++
+--
+(query object) Query you wish to run on nested objects in the `path`. If an
+object matches the search, the `nested` query returns the root parent document.
+
+You can search nested fields using dot notation that includes the complete path,
+such as `obj1.name`.
+
+Multi-level nesting is automatically supported, and detected, resulting in an
+inner nested query to automatically match the relevant nesting level, rather
+than root, if it exists within another nested query.
+--
+
+`score_mode` (Optional)::
++
+--
+(string) Indicates how scores for matching child objects affect the root
+parent document's <<query-filter-context,relevance score>>. Valid values are:
+
+`avg` (Default)::
+Use the mean relevance score of all matching child objects.
+
+`max`::
+Uses the highest relevance score of all matching child objects.
+
+`min`::
+Uses the lowest relevance score of all matching child objects.
+
+`none`::
+Do not use the relevance scores of matching child objects.
```

Comment:
Not sure, if it is worth to mention that scores of the parent objects will be equal to `0` in this case.

Reply:
Thanks @mayya-sharipova. I think that's worth mentioning. Added with fd1d6b9.